src/scss/components/switch/mixins/_hxSwitchControl.scss

```diff
@@ -9,38 +9,49 @@
 
     &::-ms-check {
       border-color: $cyan-700;
+      border-color: var(--hxSwitchControl-focus-border-color, $cyan-700);
     }
   } @else if $pseudo-state == hover {
     @if $checkBox-state == unchecked {
       background-color: $gray-0;
+      background-color: var(--hxSwitchControl-hover-unchecked-bgcolor, $gray-0);
       border: 0.125rem solid $cyan-500;
+      border: 0.125rem solid var(--hxSwitchControl-hover-unchecked-border-color, $cyan-500);
 
       &::before {
         background-color: $gray-0;
+        background-color: var(--hxSwitchControl-hover-unchecked-before-bgcolor, $gray-0);
         border: 0.125rem solid $cyan-500;
+        border: 0.125rem solid var(--hxSwitchControl-hover-unchecked-before-border-color, $cyan-500);
         left: -0.12rem;
       }
 
       &::after {
         color: $cyan-500;
+        color: var(--hxSwitchControl-hover-unchecked-after-color, $cyan-500);
       }
     } @else {
       background-color: $cyan-500;
+      background-color: var(--hxSwitchControl-hover-checked-bgcolor, $cyan-500);
```

Comment:
Please use updated colors to replace `$cyan-XXX`.

Reply:
Changes have been done, please review.